samples/opencl/FindOpenCL.cmake

```diff
@@ -0,0 +1,174 @@
+# Find OpenCL
```

Comment:
It is not public Khronos resource (it is dead private account).
Link: https://github.com/KhronosGroup/webcl-validator/blob/master/cmake/FindOpenCL.cmake